CONTRIBUTING.md

```diff
@@ -10,41 +10,91 @@ in the community are more likely to value and accept your contributions.
 ## LibreFoodPantry
 
 This project is a [LibreFoodPantry](https://librefoodpantry.org) project.
-We are a member of this larger community and shares its resources
-and policies including its
+We are a part of this larger community and shares its policies and resources.
```

Comment:
I suggest "values, policies, and resources."

Reply:
And "share", not "shares".

Reply:
Thanks! Fixing.